src/bd/settings/VmSettings.java

```diff
@@ -0,0 +1,13 @@
+package bd.settings;
```

Comment:
Yeah, I am not yet sure what to do about this.
This isn't ideal, but, the specializer somewhat depend on the setting for the DynamicMetric tool.

There is a todo, to think about better solutions. One option would be to have an interface for the context, and have the setting there. Either way, it needs additional infrastructure, which somewhat is accidental complexity for this PR, but needed to work in the context of SOMns.